bin/generate_migration

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Generating migration. Description: (e.g. 'Create admin users')"
+migrations_path="$(realpath $(dirname $0)/../src/main/resources/db/migration)"
+read description
+
+filename="V$(date +%Y.%m.%d.%H.%M.%S)__$(echo $description | sed -E 's/[^A-Z]+/_/ig').sql"
+echo "  Creating ${filename}..."
+touch "$migrations_path/$filename"
+
+echo "  Hit enter to open."
+read
+open -a "IntelliJ IDEA.app" "$migrations_path/$filename"
```

Comment:
??? Did you copy this from the FFL? They copied this originally from us. We already have it, named `bin/generate-migration` (with a dash).